.gitignore

```diff
@@ -1,4 +1,5 @@
 *.html
 target
 download
-.DS_STORE
+scripts
```

Comment:
Is this a part of your local setup you're trying to keep out-of-tree?

Reply:
I had an xde reset thing I was using. I'll remove this.